concordium-consensus/README.md

```diff
@@ -16,7 +16,7 @@ The global state depends on the lmdb library, which you may need to install usin
   ```
 - On Linux, install the `liblmdb-dev` package (or a similarly named in your
   package manager).
-- On OS X, use `brew install lmdb`.
+- On OS X, use `brew install lmdb`. In case it is not linked by default provide the flags `--extra-lib-dirs=/opt/homebrew/Cellar/lmdb/<VERSION>/lib` and `--extra-include-dirs=/opt/homebrew/Cellar/lmdb/<VERSION>/include` to the `stack build` target.
```

Comment:
An alternative would be to make a troubleshoot session in the bottom of the readme, and include the underlying section. 

This would reduce the amount of noise that sometimes tends to creep into a readme.

Reply:
This is fine.